config/eslint.js

```diff
@@ -1,13 +0,0 @@
-// We need to specify the import/resolver setting in an extendable
```

Comment:
This file was added in [this commit](https://github.com/Shopify/quilt/commit/6b3794df2de138c9aeadc64cb93d3c429beeeca0) but then became unused in [this commit](https://github.com/Shopify/quilt/commit/857890c91dda4fa9b36657dbdc8364bdb329c9c6). AFAIK it's no longer necessary?